### ISSUE TITLE
_process_incoming_event crashes when processing events with no "msg" fields.

### ISSUE THREAD

**Issue by dborry** [hynek-urban/rocketchat-async#5]
When trying to connect to my Rocket.Chat's wss, I noticed the `_process_incoming_event` always checks for a `msg` field in received events, but right after starting a session the RC server also sends this event:

```json
{"server_id":"0"}
```

This causes a crash at this line because it doesn't check that the response has a `msg` field:
https://github.com/hynek-urban/rocketchat-async/blob/master/rocketchat_async/dispatcher.py#L55

**Comment by hynek-urban** [comment]
Hi @dborry ,

thanks for reporting this. I can't reproduce this, though... what is the version of rocketchat you are using? And how are you deploying this?

Looking at [the documentation](https://developer.rocket.chat/reference/api/realtime-api), it seems that the event should always contain a `msg` field - and it works that way on my end. Googling for this, I found a few messages mentioning the `{"server_id": "0"}` format from years ago - could it be that you're using some old version of rocket.chat?

**Comment by hynek-urban** [comment]
Closing this since it can't be reproduced and there are no further details.

**Comment by computerquip** [comment]
Hi! I came across this while making a quick bot. I just figured I'd mention that this is reproducible on wss://open.rocket.chat/websocket 

The only place I see this in Rocket.Chat code is here: https://github.com/RocketChat/Rocket.Chat/blob/6acbadc96c22b6d87fb5f6e89429f85ccc3a9345/ee/apps/ddp-streamer/src/Client.ts#L118

I'm not familiar with the Rocket.Chat codebase so I'm not sure what this implies or what the `ee` thing even is, sorry. I ended up not using this module because of that.
